roles/common/templates/etc/serverspec/kvm_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
```

Comment:
Why is a KVM server spec going into the common role? There are systems that get the common role which are not compute nodes, thus wouldn't have KVM bits running. Perhaps this was meant for the nova-data role?